storm-core/src/jvm/org/apache/storm/trident/operation/builtin/Debug.java

```diff
@@ -40,7 +42,7 @@ public Debug(String name) {
 
     @Override
     public boolean isKeep(TridentTuple tuple) {
-        System.out.println(name + tuple.toString());
+        System.out.println("<"+new Date()+"> "+name + tuple.toString());
```

Comment:
@harshach If debug logs are always thrown out to system out then we can change this to use log without any backward compatibility issue. I was not sure whether debug logs are always thrown out to system.out and did not change the existing code.